src/dotnet-watch/Internal/ProcessRunner.cs

```diff
@@ -95,29 +95,38 @@ private Process CreateProcess(ProcessSpec processSpec)
             }
         }
 
-        private class ProcessState : IDisposable
+        private sealed class ProcessState : IDisposable
```

Comment:
Any reason for sealing private classes now?

Reply:
IIUC it's a very minor jit/clr perf improvement, but I'm not completely sure. I've been asked by others to add 'sealed' to private classes in other places, so I assume there are reasons for it.